cmd/storagenode/main.go

```diff
@@ -136,6 +136,18 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		return errs.New("Error starting master database on storagenode: %+v", err)
 	}
 
+	// Sync routing table database every 1s
+	ticker := time.NewTicker(time.Second)
```

Comment:
Do we have handling for corruption.

Reply:
I agree this part looks out of place.  I would prefer this syncing code be in the same place where `db.NoSync = true` is set so that it makes sense why we need this.  

I don't know the best way to add the `NoSync` setting and its related logic.  Seems like its  messy here and Im not sure if there is a way to avoid that, other than removing the NoSync code altogether and letting BoltDB handle the syncing. That is my preferred option, let BoltDB do the sync.